src/plugins/action/action.js

```diff
@@ -12,42 +12,37 @@ TemplatePath
 	});
 
 class Action extends Component {
-	constructor() {
+	constructor($scope) {
 		super(...arguments);
-	}
-
-	execute() {
-		return this.command && this.command.execute();
-	}
 
-	canExecute() {
-		return this.command && this.command.canExecute();
+		this.$scope = $scope;
 	}
 
 	onInit() {
-		const model = this.model;
-		const action = new ActionItem(this.command, this.title, this.icon);
-		action.id = this.id;
-
-		const actions = Array.from(model.action().items);
-		actions.push(action);
+		const context = {
```

Comment:
u can make like `ActionView(this.model, this)` and it will persist bindings

Reply:
I'm not sure that it's good from typification perspective